### PR TITLE
Add ability to save stall and item reviews

### PR DESCRIPTION
**Pull request by samuelmui8**

### CODE REVIEW

File: src/main/java/seedu/address/model/stall/Stall.java [hunk 1]
```diff
@@ -52,6 +52,10 @@ public Menu getMenu() {
         return menu;
     }
 
+    public boolean hasStallReview() {
```
Comment (teoks0199) [inline src/main/java/seedu/address/model/stall/Stall.java:55]: maybe Stall should have an overloaded constructor that can take in Reviews as arguments too

**Review by teoks0199 — APPROVED** [review]
lgtm

**Review by teoks0199 — APPROVED** [review]
lgtm

File: src/main/java/seedu/address/storage/JsonSerializableMenu.java [hunk 2]
```diff
@@ -28,7 +28,9 @@ public class JsonSerializableMenu {
      */
     @JsonCreator
     public JsonSerializableMenu(@JsonProperty("items") List<JsonAdaptedItem> items) {
-        this.items.addAll(items);
+        if (items != null) {
```
Comment (Ruizhi2001) [inline src/main/java/seedu/address/storage/JsonSerializableMenu.java:31]: null check

File: src/test/java/seedu/address/storage/JsonAdaptedStallTest.java [hunk 3]
```diff
@@ -34,14 +34,14 @@ public void toModelType_validStallDetailsWithItems_returnsStall() throws Excepti
     @Test
     public void toModelType_invalidName_throwsIllegalValueException() {
         JsonAdaptedStall stall =
-                new JsonAdaptedStall(INVALID_NAME, VALID_LOCATION, null);
+                new JsonAdaptedStall(INVALID_NAME, VALID_LOCATION, null, null);
```
Comment (Ruizhi2001) [inline src/test/java/seedu/address/storage/JsonAdaptedStallTest.java:37]: good testcases to cover all constructors

**Review by Ruizhi2001 — APPROVED** [review]
lgtm

**Pull request merged**